helpers/load_test_interface.go

```diff
@@ -174,8 +175,19 @@ func WRK2LoadTest(opts *models.LoadTestOptions) (map[string]interface{}, *period
 func startNighthawkServer(timeout int64) error {
 	nighthawkStatus.Lock()
 	defer nighthawkStatus.Unlock()
-	command := "./nighthawk_service"
-	transformCommand := "./nighthawk_output_transform"
+
+	curDir, err := os.Getwd()
+	if err != nil {
+		return ErrStartingNighthawkServer(err)
+	}
+
+	command := filepath.Join(curDir, "nighthawk_service")
+	transformCommand := filepath.Join(curDir, "nighthawk_output_transform")
+
+	_, err = os.Stat(command)
+	if err != nil {
+		return ErrStartingNighthawkServer(err)
```

Comment:
Should we deviate from MeshKit-style errors?

Reply:
Sir, this is a MeshKit error defined here https://github.com/meshery/meshery/blob/d21b9d0f9e7ea40dfc666a325b4cd31e4e98dc4d/helpers/error.go#L131
:stuck_out_tongue:

Reply:
I refuse to be spoken to this way. Your slight corrections and polite edifications are.... well... they just leave me no room to piss and moan and... I'm just not sure what to do when this sort of thing happens.